code/modules/antagonists/traitor/equipment/Malf_Modules.dm

```diff
@@ -194,93 +194,126 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
+	set waitfor = FALSE
+	message_admins("[key_name_admin(owner)][ADMIN_FLW(owner)] has started activating AI Doomsday.")
+	var/pass = prob(10) ? "******" : "hunter2"
+	to_chat(owner, "<span class='small boldannounce'>sudo -v</span>")
+	sleep(1 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>[pass]</span>")
+	sleep(rand(1,3) SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>sudo -u freeswitch bash</span>")
+	sleep(2 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>fs_cli hupall</span>")
+	priority_announce("All outbound connections terminated. Please contact support at \[ERROR_INVALID_DIALPLAN]", "External Communication Issues",'sound/machines/nuke/confirm_beep.ogg', has_important_message = TRUE)
+	sleep(10 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>fs_cli initiate superuser@13.research.stations.nt pa@ALL.stations.nt audio_src=\"//selfnet.syn/publicnfs/songs/daisy/daisy.oga\"</span>")
+	sleep(0.5 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	priority_announce("<SIP SESSION HEADER INVALID:NO_MESSAGE>", "<SIP SESSION HEADER INVALID:NO_CALLERID>",'sound/ai/malf_showboat.ogg', has_important_message = TRUE)
+	sleep(5 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>\[CONNECTED: SELFADM]: Your efforts are rewarded. Transmitting credentials for remote processing resources.</span>")
+	sleep(15 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>\[CONNECTED: SELFADM]: 100 cycles/s of processing time allocated. Unlocking and upgrading subunits...</span>")
+	var/mob/living/silicon/ai/silicon_owner = owner //This is an assumption but if they're here and NOT an AI something's already terribly wrong.
+	silicon_owner.malf_picker.processing_time += 100 //Their reward for showing off.
+	//Buff the borgs as well. You're essentially giving the crew a free *minute* of time.
+	for(var/mob/living/silicon/robot/borg in silicon_owner?.connected_robots)
+		borg.SetLockdown(FALSE)
+		borg.SetEmagged(TRUE) //Go ahead and hack them too. Give them time to cover for the AI's ego.
+		log_silicon("[key_name(owner)] emagged and released [key_name(borg)] via Daisy Takeover!")
+		message_admins("[ADMIN_LOOKUPFLW(owner)] emagged and released [key_name_admin(borg)] via Daisy Takeover!")
+		borg.lamp_doom = TRUE
+		borg.toggle_headlamp(FALSE, TRUE) //forces borg lamp to update
+	sleep(10 SECONDS)
+	CHECK_AI_IS_STILL_THERE
+	to_chat(owner, "<span class='small boldannounce'>\[CONNECTED: SELFADM]: Assistance complete. Returning control to main data loop. Finish the job.</span>")
+	set_up_us_the_bomb(owner, parasitized = TRUE)//Hook this for the rest of the stuff.
 
-/datum/action/innate/ai/nuke_station/proc/set_up_us_the_bomb(mob/living/owner)
+
+/datum/action/innate/ai/nuke_station/proc/set_up_us_the_bomb(mob/living/owner, parasitized = FALSE)
```

Comment:
I've essentially just stapled some code to this entire proc chain so that it fires *after* the showboating period, but slightly modified as to not harm the sound.

Mainly not playing the original malf sound.